### PR TITLE
Update k8s data-governance.mdx

### PR DESCRIPTION
**Pull request by wadhah-nr**
## Give us some context

Enriching k8s data governance documentation by adding ways that would enable users to optimise and govern logs collected as well as separate the content based on the data type (metrics and logs).

### CODE REVIEW

**Comment by akristen** [comment]
Per our SLAs, this will be reviewed + merged within the next 3 days! I'll follow up with any questions as they come up in review. Thanks a bunch for submitting this PR!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx [hunk 1]
```diff
@@ -7,7 +7,151 @@ tags:
...
+The collection of  [Logs](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L27) in Kubernetes is  done  using the [New Relic kubernetes plugin](https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/), an output plugin for Fluentbit.
 
+The configuration of **fluentbit** includes:
+- **Input Plugin**: This is where we define the log sources that we wish to collect, as in our case we are talking about k8s logs.
+- **Parser Plugins**: to convert and structure log messages (JSON, Regexp, LTSV, Logfmt, etc.)
+- **Filter Plugins**: to modify, enrich and delete   information from your logs (Nest, Throttle, Expect, Geoplp, Grep, K8s, etc.) For filtering, you can also use a LUA script.
+- **Output Plugins**: send the log stream to several outputs which will be responsible for storing the log stream and displaying them. In this case, we speak of NRDB (New Relic Database).
+
+Logs can be filtered following the methods below:
+### Change the path
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:20]: @wadhah-nr The "Change path" / "Exclude paths" / "Pod annotations" 3 separate procedures customers choose from, or is it a 3 step procedure that customers need to work through sequentially?

Comment (wadhah-nr) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:20]: @akristen yes exactly: those represent 3 different ways/options for customers to optimise log data ingestion.

Comment (akristen) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:20]: @wadhah-nr When do we recommend a certain method to certain customers? We like to give some guidance on when a customer's use case might fit one method over another. In other words, when might a customer choose:

* Changing the path over the other two?
* Excluding the path over the other two?
* Using pod annotations?

Comment (akristen) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:20]: A follow up question: Does New Relic prefer one method over all other use cases? if so, why?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx [hunk 2]
```diff
@@ -7,7 +7,151 @@ tags:
...
+In the input configuration, we have the **path** defined, which would point the fluentbit agent to the **log paths** that need to be collected.
+
+For k8s containers, logs are stored in files with the format: **"/var/log/containers/<podname>_<namespace>....log"** (file example: **/var/log/containers/nri-bundle-newrelic-logging-pjdfks_default_newrelic-logging-1d45d23135d7062c1fcd7c872472a74d5294c55edd0592a2130c10b046234.log**)
+
+If you would like to concentrate for example the collection of logs on 3, 4 pods: you would have to specify the paths according to the pods names and the wildcard  (*):
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:25]: (Follow up question: What is the default behavior? To collect all log data from all pods?)

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx [hunk 3]
```diff
@@ -7,7 +7,151 @@ tags:
...
+
+```yaml
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:27]: So they're changing the path in a Fluent Bit plugin file? Can we tell customers where they might find this file, or does it vary from person to person?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx [hunk 4]
```diff
@@ -7,7 +7,151 @@ tags:
...
+fluentBit:
+  logLevel: "info"
+  path: "/var/log/containers/myPod1*.log, /var/log/containers/myPod2*.log"
+```
+
+So it ends up looking like:
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx:33]: what ends up looking like? Can you unspool how the first fluentbi yaml file leads to this global config? What's the connection between the two?

**Comment by akristen** [comment]
Closing this until SME has capacity to work on this PR. can easily reopen and add it to a sprint when time permits!

**Pull request closed without merging**